scripts/vagrant/zeppelin-dev/roles/maven/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+# setup maven
+# Assumes Ubuntu Server 14.04 LTS (HVM)
+# Assumes Java 7+ jdk installed
+---
+- name: download maven
+  get_url: url=http://apache.mirrorcatalogs.com/maven/maven-3/3.3.3/binaries/apache-maven-3.3.3-bin.tar.gz
```

Comment:
@Leemoonsoo I recall you have a script to handle the mirror page - I don't seem to be able to find it.
Alternatively you could check out https://github.com/y-higuchi/apache-mirror-selector